configuration/pih/liquibase/sql/create_functions.sql

```diff
@@ -1755,3 +1755,31 @@ BEGIN
 
 END
 #
+-- This function acceps patient id, a list of encounter types, visit id, form id and a begin date
+-- it will return the latest encounter for that patient, of one of those encounter types, in that visit of that form type since that date
+-- form id and begin date can be left null
+#
+DROP FUNCTION IF EXISTS latestEncForminVisit;
+#
+CREATE FUNCTION latestEncForminVisit(_patientId int(11), _encounterTypes varchar(255), _visitId int(11), _formId int(11), _beginDate datetime)
```

Comment:
We should be a little careful of referencing encounter types by name instead of UUID, but guess the names are fairly stable.

I was suprised/had forgotten that we had used the same encounter type for Consult and ED Triage forms...

Reply:
Nice catch.  I meant to go back and replace the code that I used to call with a parameter I load in from the encounter type table.  I will make this change and commit!